### ISSUE TITLE
Regarding the issue of a large amount of code being commented out in dataloder.py

### ISSUE THREAD

**Issue by zzzmxxxx** [XpastaX/ConFEDE#17]
Hello author, may I ask if many methods defined in your dataloder.py have not been used, such as normalization or adding masks. Even some of the methods used have a large amount of code commented out. What are the functions of these methods? I hope to receive your reply, thank you.

**Comment by zzzmxxxx** [comment]
What I mean is that neither data filling nor normalization methods have been used. Was they originally intended to be needed?

**Comment by XpastaX** [comment]
Sorry for the misleading codes. Those are just codes for early development that I forgot to remove. For example:
```       
 self.rank_M = torch.zeros(self.cos_matrix_M.shape)
# self.rank_T = torch.zeros(self.cos_matrix_T.shape)
# self.rank_V = torch.zeros(self.cos_matrix_V.shape)
# self.rank_A = torch.zeros(self.cos_matrix_A.shape)
```
I tried to use separate modality features for ranking and found it has no improvements and could be too complicated. So I decided to use the concatenated version.
